docs/guidelines/content/unwelcome-concepts.md

```diff
@@ -54,7 +54,7 @@ When these are acceptable:
 
 ## Requires Complete Perfection
 
-Achievements that require a long task where the slightest mistake will end in failing the challenge. Especially challenges where randomness and other variation in game behavior makes anticipating actions extra difficult.
+Achievements that require a **long task** where the slightest mistake will end in failing the challenge. Especially challenges where randomness and other variation in game behavior makes anticipating actions extra difficult.
```

Comment:
Recommend also quantifying "long task". Can we provide any examples that would click for a reader to help reduce ambiguity?

Reply:
You're not wrong, but this is intended as a quick highlight for now until we come up with more solid improvements